feat: Add real time cryptocurrency chart

lionking917 opened the pull request.

lionking917 commented:
@Niaro 
Can you kind check again?
Sorry, I forgot to commit package.json file.
So please check now.
It definitely will work now.
Thanks for your patience.
Regards.

Niaro commented:
@lionking917 Yeap, now it works! But still, the most crucial part is missing. The line chart. Usually, we discuss the implementation details of a task solution in a technical interview, and the current implementation lacks the body. So please, if you could, provide some.

lionking917 commented:
@Niaro I removed chartJS packages and manually drew line chart using Canvas itself.
Also changed some logics too.
Please check kindly.
Thanks.
Really hope to work with you.
Warm regards.
